OrbitLinuxTracing/PerfEventOpen.cpp

```diff
@@ -114,7 +114,7 @@ int uretprobes_event_open(const char* module, uint64_t function_offset,
 void* perf_event_open_mmap_ring_buffer(int fd, uint64_t mmap_length) {
   // The size of the ring buffer excluding the metadata page must be a power of
   // two number of pages.
-  if (mmap_length < getpagesize() ||
+  if (mmap_length < static_cast<size_t>(getpagesize()) ||
```

Comment:
If I understand correctly, we should replace getpagesize() with sysconf(_SC_PAGESIZE); see http://man7.org/linux/man-pages/man2/getpagesize.2.html.

Reply:
Yes, I read that too, but I didn't want to make a logical change here. As far as I understood it the problem does not occur on x86 but on other platforms (The doc mentions SPARC).